app/models/principal.rb

```diff
@@ -6,11 +6,13 @@ class Principal < ActiveRecord::Base
 
   has_one :firm, primary_key: :fca_number, foreign_key: :fca_number
 
+  validates_presence_of :fca_number
```

Comment:
What was the rationale for breaking this out? As far as I know, you can keep it in with the `validates` and `allow_blank` will work as expected.